research/make_model.py

```diff
@@ -0,0 +1,60 @@
+"""
+    Allows for the preprocessing a dataset
+    of airport data; and using it to then make, 
+    train, and save a Random Forest Regression model. 
+"""
+
+import pickle
+import json
+from sklearn.ensemble import RandomForestRegressor #type: ignore [import-untyped]
+import pandas as pd
+from data.json_converter import DataConverter
+from data.stats.get_runway_stats import RunwayStats
+from data.stats.get_flight_stats import FlightStats
+from src.data.types import Airport
+
+class MakeModel:
+    def __init__(self) -> None:
+        self._model = RandomForestRegressor(n_estimators=10, max_features=2)
+
+    def train_fit(self, data_filename: str, save_filename: str) -> "MakeModel":
+        with open(data_filename, "r", encoding="utf-8") as f:
+            raw_data = json.load(f)
+        target, features = self.preprocessing(raw_data)
+        self._model.fit(features,target.values.ravel())
+        self.__save_trained_model(save_filename)
```

Comment:
It's probably better to make `save` a public method and let the called decide what to do with the model once it's been trained.